### ISSUE TITLE
Missing compatibility data for aria-keyboardshortcuts

### ISSUE THREAD

**Issue by schalkneethling** [mdn/content#29817]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-keyshortcuts

### What specific section or headline is this issue about?

Browser compatibility

### What information was incorrect, unhelpful, or incomplete?

It seems that the BCD data is missing from the page.

### What did you expect to see?

I expected to see the browser compatibility table for the `aria-keyshortcuts` attribute.

### Do you have any supporting links, references, or citations?

I looked up the data from caniuse, which suggests that MDN has this data available in the BCD repo.

https://caniuse.com/?search=aria-keyshortcuts

![Screenshot suggesting that BCD does contain support documentation for this attribute.](https://github.com/mdn/content/assets/10350960/5f77a354-d663-437d-b11e-80052eb808d9)

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/accessibility/aria/attributes/aria-keyshortcuts`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-keyshortcuts
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/attributes/aria-keyshortcuts/index.md
* Last commit: https://github.com/mdn/content/commit/940b38ca51895641529ea1c72842f1a2d6fe987a
* Document last modified: 2023-09-04T14:48:57.000Z

</details>

**Comment by caugner** [comment]
Possibly related: https://github.com/mdn/mdn/issues/389

**Comment by caugner** [comment]
@schalkneethling Good to see you! 👋 

Note that the caniuse page refers to the _property_ [ariaKeyShortcuts](https://developer.mozilla.org/en-US/docs/Web/API/Element/ariaKeyShortcuts), not the attribute [aria-keyshortcuts](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-keyshortcuts).

It looks like browser-compat-data currently doesn't have any data for ARIA attributes, and I believe this is due to the fact that afaik browsers don't treat those ARIA attributes specially, unless a [Screen reader](https://developer.mozilla.org/en-US/docs/Glossary/Screen_reader) is used.

**Comment by Josh-Cena** [comment]
ARIA is not mainly intended for browsers. This is a much bigger project to have ARIA compatibility data.
